apps/workspaces/apis/import_settings/views.py

```
@@ -1,11 +1,30 @@
+
+
+class ImportCodeFieldView(generics.GenericAPIView):
+    """
+    Import Code Field View
+    """
+    def get(self, request, *args, **kwargs):
+        workspace_id = kwargs['workspace_id']
+        category_import_log = ImportLog.objects.filter(workspace_id=workspace_id, attribute_type='CATEGORY').first()
+
+        response_data = {
+            'ACCOUNT': False if category_import_log else True,
```

Ruff

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)